birdhouse/docker-compose.yml

```diff
@@ -212,10 +234,9 @@ services:
       - "8983:8983"
       - "48983:9001"
     volumes:
-      - /data/solr:/data
-    volumes_from:
-      - mongodb
```

Comment:
I personally never understood why Solr and ncwms2 needs volume_from mongodb.  But was afraid to change since I do not know Solr and ncwms2 well.  So you confident this do not break anything?

Reply:
I'm not really worried about ncwms2 as : 
1) we are now using thredds only
2) I don't see any reason at all why ncwms would use mongo

For Solr though, it can use mongo as backend, but could this be done using only a volumes_from? I don't know, but as Long said we never got confident enough to remove it. 

Booting is a thing, does the indexation occurs and the search works? If we now have tests to prove that, then I think we could get rid of that for good.

Reply:
I will add the `/data/solr` one back in case, but the `volumes_from` didn't do anything since `mongo` did not have any mapped volumes to start with, unless overridden in some private config somewhere.

Reply:
If the following test https://github.com/Ouranosinc/pavics-sdi/blob/e50335e76d0018cf813bec1c963bff576b881345/docs/source/notebooks/catalog_search.ipynb (thank you @MatProv to provided me the link) succeed, we will be ok (not ok right now). The fastest way to know if volumes_from is the culprit, would be to revert the change and test again. Once we found what is absolutely required, we should add a note so that we know in the future what cannot be removed.

Please also note that we are not using the stock mongodb but the one build by birdhouse buildout mecanism. Their is a recipe for solr and I looked at it trying to figured out what could link mogo and solr without any success.

Reply:
Like I said, probably by Phoenix. For Solr I don't see how, but there is a chance too.